embed resource files when the compiler supports it.

mhilton opened the pull request.

jujugui commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/78/

jujugui commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/79/

rogpeppe commented:
:shipit:

jujugui commented:
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candid-merge

mhilton committed:
format code

mhilton committed:
embed templates and static files

If the go compiler supports it embed the template and static files in
the candidsrv binary.

mhilton committed:
Merge branch 'master' into 200-embed-resources
